hack/windows/Prepare-AntreaAgent.ps1

```diff
@@ -13,7 +13,8 @@ Specifies whether kube-proxy interface is included in the installation. If false
 be installed on the host.
 #>
 Param(
-    [parameter(Mandatory = $false)] [bool] $InstallKubeProxy = $true
+    [parameter(Mandatory = $false)] [bool] $InstallKubeProxy = $true,
+    [parameter(Mandatory = $false)] [bool] $ContainerizeOVS = $false
```

Comment:
Maybe rename "ContainerizeOVS" as "RunOVSServices", and the default value is true to be consistent with the original behaviors?

Reply:
Okay